R/ResetModule.R

```diff
@@ -65,19 +68,26 @@ reset_report_button_srv <- function(id, reporter) {
             footer = shiny::tagList(
               shiny::tags$button(
                 type = "button",
-                class = "btn btn-secondary",
-                `data-dismiss` = "modal",
+                class = "btn btn-outline-secondary",
                 `data-bs-dismiss` = "modal",
                 NULL,
                 "Cancel"
               ),
-              shiny::actionButton(ns("reset_reporter_ok"), "Reset", class = "btn-danger")
+              shiny::actionButton(ns("reset_reporter_ok"), "Reset", class = "btn btn-outline-primary")
             )
           )
         )
       )
     })
 
+    observeEvent(reporter$get_reactive_add_card(), {
+      if (reporter$get_reactive_add_card() > 0) {
+        shinyjs::enable(id = "reset_reporter")
+      } else {
+        shinyjs::disable(id = "reset_reporter")
+      }
+    })
+
```

Comment:
Duplicate (please look 30 lines before)

Reply:
Thanks, I will replace `shinyjs::enable/disable` with `shinyjs::toggleClass` in all the places.